src/main/java/com/facebook/openwifirrm/DeviceDataManager.java

```diff
@@ -550,13 +551,37 @@ public void setDeviceApConfig(String serialNumber, DeviceConfig apConfig) {
 	}
 
 	/** Device AP config layer update interface. */
-	public interface DeviceApConfigFunction {
+	public interface LayerConfigFunction {
```

Comment:
Instead of renaming this, can you make a separate interface instead for AP vs. zone? It barely duplicates any code and will make it clearer (ex. when searching). Also, the same interface method wouldn't work for updating the network layer.